src/ansys/aedt/core/modeler/cad/primitives_3d.py

```diff
@@ -1464,6 +1464,56 @@ def _create_reference_cs_from_3dcomp(self, assignment, password):
             app.oproject.Close()
             return assignment.target_coordinate_system
 
+    @staticmethod
+    def __create_temp_project(app):
```

Comment:
👍 for naming this static methods as "private". At first I though it was a mistake as the "usual" behavior of static method is to be public but here they are only used internally. Glad you did this the way you did it !

Reply:
Indeed, we should start migrating some similar method to private, step by step :)